src/build-release-graph.js

```diff
@@ -10,6 +10,8 @@ const { trackNewVersion } = require('./version');
 const semver = require('semver');
 const dependencyTypes = require('./dependency-types');
 
+const defaultReleaseType = 'patch';
+
 async function getReleaseType(packageName, cwd) {
```

Comment:
Now this needs to be exported and the export is being used within the file is it worth breaking it out into a separate file? 
I think it would make the code more readable.

Reply:
I could, but I would still need to call `module.exports.getReleaseType` in this file. That is how I spy the call in the tests.